client/my-sites/earn/launchpad/index.tsx

```diff
@@ -58,7 +32,7 @@ const EarnLaunchpad = ( {
 					{ translate( 'Let your fans support your art, writing, or project directly.' ) }
 				</p>
 			</div>
-			<Launchpad siteSlug={ site?.slug ?? null } checklistSlug="earn" taskFilter={ taskFilter } />
+			<Launchpad siteSlug={ site?.slug ?? null } checklistSlug="earn" />
```

Comment:
Removing taskFilter here is the main change. All other code is removed because it was needed to add this.

Reply:
Can you try importing and using the `DefaultWiredLaunchpad` component instead? This is the wrapping component from #82612 that means you should be able use the defaults and move on without much boilerplate.

Reply:
@daledupreez - Thanks! That's been updated and works as expected now.